tests/integration/test_pileup.py

```diff
@@ -0,0 +1,221 @@
+import requests
+import dash
+import dash_bio
+import dash_html_components as html
+import os
+import re
+
+import time
+from common_features import simple_app_layout
+
+_COMPONENT_ID = 'mypileup'
+
+_GENOME = "hg19"
+_GEAR_ICON = '⚙'
+
+def test_dbpileup001_reference(dash_duo):
```

Comment:
```suggestion
def test_dbpu001_reference(dash_duo):
```

Just a minor change to standardize the test name to the other integration tests.